src/main/java/com/moddy/server/service/offer/HairServiceOfferRetrieveService.java

```diff
@@ -116,13 +117,23 @@ private List<OfferResponse> getModelMainOfferList(final Page<HairServiceOffer> o
                 return offerCondition.getOfferCondition().getValue();
             }).collect(Collectors.toList());
 
-            OfferResponse offerResponse = new OfferResponse(offer.getId(), designer.getProfileImgUrl(), designer.getName(), designer.getHairShop().getName(), offerConditionTop2List, offer.isClicked());
+            OfferResponse offerResponse = new OfferResponse(offer.getId(), designer.getProfileImgUrl(), designer.getName(), designer.getHairShop().getName(), offerConditionTop2List, calOfferStatus(offer));
             return offerResponse;
         }).collect(Collectors.toList());
 
         return offerResponseList;
     }
 
+    private OfferStatus calOfferStatus(final HairServiceOffer hairServiceOffer){
+        if(hairModelApplicationRetrieveService.getApplicationExpiredStatus(hairServiceOffer.getHairModelApplication().getId())){
+            return OfferStatus.EXPIRED;
+        } else if (hairServiceOffer.isClicked()){
+            return OfferStatus.CLICKED;
+        } else {
+            return OfferStatus.UNCLICKED;
+        }
+    }
```

Comment:
```suggestion
    private OfferStatus calOfferStatus(final HairServiceOffer hairServiceOffer){
        if(hairModelApplicationRetrieveService.getApplicationExpiredStatus(hairServiceOffer.getHairModelApplication().getId())){
            return OfferStatus.EXPIRED;
        } else if (hairServiceOffer.isClicked()){
            return OfferStatus.CLICKED;
        }
        return OfferStatus.UNCLICKED;
    }
```
p4
마지막 else 문을 없애서 들여쓰기를 하나 없앨 수 있을 것 같습니다...!